services/generic/virtual_meter.md

```diff
@@ -1,64 +1,84 @@
-### Virtual meter service
+# Virtual Meter Service
 
-This service enables a device to report accumulated consumption and/or pulses in case it does not have its own metering capabilities (eg. a relay with output binary switch service or thermostat accumulating momentary power measured in Watts into energy consumption measured in kWh). If a virtual meter service is present in a device, upon manual configuration providing a consumption per supported unit, a proper meter service will be added to the device. The service then will calculate cumulated consumption at every state change, mode change and at least once during a configured interval.
+This service enables a device to report an accumulated consumption and/or pulses in case it does not have its own metering capabilities, e.g. relay with output binary switch
+service or thermostat. For example a virtual electricity meter works by accumulating momentary power measured in Watts into lifetime energy consumption
```

Comment:
Remove "For example"

Reply:
Okies, but this for example was to foreshadow possibly of more than one virtual meter service.